packages/CLAPACK/meta.yaml

```diff
@@ -17,7 +17,7 @@ source:
     - [make.inc, make.inc]
 
 build:
-  library: true
+  sharedlibrary: true
```

Comment:
Could you please add it to https://github.com/iodide-project/pyodide/blob/master/docs/development/new-packages.md#the-metayaml-file